modules/test/playwright/tests/commerce/utils/date.ts

```diff
@@ -74,22 +83,26 @@ export function checkSameDate(dateFromApi: string, date2: string) {
 	const firstDate = new Date(dateFromApi);
 	const secondDate = new Date(date2);
 
-	return firstDate.getUTCDate() === secondDate.getDate() &&
-		   firstDate.getUTCMonth() === secondDate.getMonth() &&
-		   firstDate.getUTCFullYear() === secondDate.getFullYear() &&
-		   firstDate.getUTCHours() === secondDate.getHours() &&
-		   firstDate.getUTCMinutes() === secondDate.getMinutes();
+	return (
+		firstDate.getUTCDate() === secondDate.getDate() &&
```

Comment:
Are you sure you can compare `getUTC[Month|Hours|...]()` with `get[Month|Hours|...]`?